work/v1/types.go

```diff
@@ -79,9 +79,55 @@ type ManifestConfigOption struct {
 	ResourceIdentifier ResourceIdentifier `json:"resourceIdentifier"`
 
 	// FeedbackRules defines what resource status field should be returned.
+	// +optional
+	FeedbackRules []FeedbackRule `json:"feedbackRules,omitempty"`
+
+	// UpdateStrategy defines the strategy to update this manifest
+	// optional
+	UpdateStrategy UpdateStrategy `json:"updateStrategy"`
```

Comment:
to make this work how you want as optional, I think you it to be a pointer and be omitempty.